projects/igniteui-angular/src/lib/tabs/tab-item.component.ts

```diff
@@ -25,13 +27,53 @@ export class IgxTabItemComponent implements IgxTabItemBase {
     @Input()
     public relatedGroup: IgxTabsGroupComponent;
 
+    /**
+    * An @Input property that sets the value of the `icon`.
+    * The value should be valid icon name from {@link https://material.io/tools/icons/?style=baseline}.
+    *```html
+    *<igx-tab-item label="Tab 1" icon="home">
+    *```
+    */
+    @Input()
+    public icon: string;
```

Comment:
Consider getting/setting the relatedGroup.label and relatedGroup.icon when the tabs have relatedGroup. This will make the context bindings in the template unnecessary.

Reply:
Addressed.